layers/meta-resin-raspberrypi/conf/samples/bblayers.conf.sample

```diff
@@ -14,7 +14,7 @@ BBLAYERS ?= " \
     ${TOPDIR}/../layers/meta-openembedded/meta-python \
     ${TOPDIR}/../layers/meta-raspberrypi \
     ${TOPDIR}/../layers/meta-resin/meta-resin-common \
-    ${TOPDIR}/../layers/meta-resin/meta-resin-pyro \
+    ${TOPDIR}/../layers/meta-resin/meta-resin-rocko \
```

Comment:
This needs a change bellow aswell in line 26

Reply:
Yes. We can address this when we include meta-resin submodule update too.

Reply:
Done.